### DISCUSSION TITLE
Multiple "per-application" backups in the same repository?

### DISCUSSION THREAD

**Discussion by szpak** [borgbase/vorta#1941] [FAQ] [answered]
I really like Borg, but for non-technical family members shell commands might be challenging :). I've bumped into Vorta and it looks very promising!

However, I would like to be able to set different retention policies (prune configuration) for certain "applications" (in fact different "types of files", such as: Documents, Downloads, Music, etc). To leverage deduplication, I would like to put them all into a single Borg repository (just with different archive names).

I wonder, if it is supported/recommended in Vorta to achieve the aforementioned with different profiles (precised different source paths and different retention policies) pointing out to the same Borg repository? Or it should be rather handled in some other way?

We can assume that manual backup execution is enough (so need to automatically trigger backups from different profiles).

**Comment by m3nu** [comment, marked as answer]
Have you tried making multiple profiles for each file type? Those can use the same repo and have different settings for source folders, prefix and pruning. Setting different prefixes will be key. Like `{hostname}-documents-...`

**Reply by szpak** [reply]
I'm just at the documentation reading stage and the profiles were my first idea, but I asked that question to get know if there are any know consequences. From your answer, I assume that no and I will give it a try in practice. Thanks.

**Reply by szpak** [reply]
Probably I missed something, but while `new_archive_name` is configurable:
https://github.com/borgbase/vorta/blob/b2cf5b1fc9e6758b0d7a2e79a03d6740bee74826/src/vorta/store/models.py#L100

is there any way set the new value in GUI? Or it is required to edit the database (or easier - to export to JSON, replace `new_archive_name` and import)?

**Reply by szpak** [reply]
Btw, for people hitting this thread in the future, also `prune_prefix` likely needs to be changed (in addition to `new_archive_name`).

Btw2, that could make the UI somehow over-bloated. Nevertheless, maybe there could be some "advanced settings" dialog with a list of available settings to edit in a table (such as in Firefox or Thunderbird)?

**Reply by szpak** [reply]
> Those can use the same repo and have different settings for source folders, prefix and pruning. Setting different prefixes will be key. Like {hostname}-documents-...

Btw3, It works as expected, thanks.

**Reply by szpak** [reply]
I [created](https://github.com/borgbase/vorta/issues/1948) a related future request.
